shared/selectNetworkConfig/artik710.md

```diff
@@ -0,0 +1,14 @@
+Before your resinOS download begins, a prompt will appear asking you to specify which version of resinOS you want to run and how your device will connect to the internet. 
+Make sure to select `Resin OS 2.0.2-beta.2` from the dropdown menu.
```

Comment:
@craig-mulligan there is an issue, where there is no recommended, its just a blank drop down, this image will become the recommended when that bug is fixed